packages/bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -70,7 +73,12 @@ const getIconForType = type => {
   return map[type];
 };
 
-const ToggleButton = props => {
+type ToggleButtonProps = {
+  label: ?string,
```

Comment:
This was previously required, but the prop in the outer component wasn't so this is a clarification of what should happen here. i.e this was a bug. We should consider if we want to handle this differently